eng/pipelines/common/templates/runtimes/build-runtime-tests.yml

```diff
@@ -0,0 +1,24 @@
+parameters:
+  osGroup: ''
+  osSubgroup: ''
+  archType: ''
+  buildConfig: ''
+  liveLibrariesBuildConfig: ''
+  testBuildArgs: ''
+  #arcade-specific parameters
+  condition: always()
+  continueOnError: false
+  displayName: ''
+  timeoutInMinutes: ''
+  enableMicrobuild: ''
+  gatherAssetManifests: false
+  shouldContinueOnError: false
```

Comment:
If the template doesn't seem to use these, do we need to define them?

Reply:
We can remove them, but I was keeping consistency with our other templates. In another PR I’ll go through and clean up some of these across all of our extra-steps templates and make the parameter lists consistent.